lib/page-data/actions.js

```diff
@@ -220,13 +220,13 @@ export function schedulePage(store, { uri, timestamp }) {
  */
 export function unschedulePage(store, publishing = false) {
   const pageUri = _.get(store, 'state.page.uri'),
-    scheduleId = btoa(pageUri),
+    scheduledId = btoa(pageUri),
     prefix = _.get(store, 'state.site.prefix'),
-    scheduleUri = `${prefix}${scheduleRoute}/${scheduleId}`;
+    scheduledUri = `${prefix}${scheduleRoute}/${scheduledId}`;
 
-  store.dispatch('startProgress', 'scheduled');
+  store.dispatch('startProgress', 'unscheduled');
 
-  return remove(scheduleUri)
+  return remove(scheduledUri)
```

Comment:
was this the main issue?

Reply:
The main issues were [here](https://github.com/clay/clay-kiln/pull/1554/files#diff-a05c20d36fd595f8e9f570e888fc5bbde303b1a4d661063d6ddd81dd49e480e9R231) and [here](https://github.com/clay/clay-kiln/pull/1554/files#diff-a05c20d36fd595f8e9f570e888fc5bbde303b1a4d661063d6ddd81dd49e480e9R331).

The first one was causing [this function](https://github.com/clay/clay-kiln/blob/155d71f09e893eea6672258e6ea4d7783aa8a299/lib/page-data/actions.js#L221) to behave completely different because of the incorrect parameter being passed and the second issue was preventing the UI from being fully updated!